lumicks/pylake/kymo.py

```diff
@@ -261,39 +261,65 @@ def plot_with_time_histogram(self, color_channel, pixels_per_bin=1, hist_ratio=0
         ax_hist.set_ylabel("counts")
         ax_hist.set_title(self.name)
 
-    def downsampled_by(self, time_factor, reduce=np.sum):
+    def downsampled_by(
+        self, time_factor=1, position_factor=1, reduce=np.sum, reduce_timestamps=np.mean
+    ):
         """Return a copy of this Kymograph which is downsampled by `time_factor`
```

Comment:
The docstring summary here needs updating, no?

Reply:
Yep, good catch!